README.md

```diff
@@ -0,0 +1,32 @@
+ArborX
+======
+
+[![Build Status](https://cloud.cees.ornl.gov/jenkins-ci/buildStatus/icon?job=ArborX-continuous)](https://cloud.cees.ornl.gov/jenkins-ci/job/ArborX-continuous/)
+[![Documentation Status](http://readthedocs.org/projects/arborx/badge/?version=latest)](http://arborx.readthedocs.io/en/latest/?badge=latest)
+[![codecov](https://codecov.io/gh/ArborX/ArborX/branch/master/graph/badge.svg)](https://codecov.io/gh/ArborX/ArborX)
```

Comment:
No badges for now (Jenkins link does not exist, doc has nothing, coverage is not setup)